phial/bot.py

```diff
@@ -328,6 +331,9 @@ def _execute_response(self,
         if isinstance(response, str):
             self.send_message(Response(text=response, channel=command.channel))
 
+        elif isinstance(response, list) and all(isinstance(x, MessageAttachment) for x in response):
```

Comment:
I don't think we should be able to return just a list of attachments, I feel they should always belong to a `Response`. Returning an attachment with having anything to 'attach' it to just feels a little off to me.